test/functional/apps/visualize/_tsvb_chart.ts

```diff
@@ -28,11 +29,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   describe('visual builder', function describeIndexTests() {
     this.tags('includeFirefox');
     beforeEach(async () => {
-      await security.testUser.setRoles([
-        'kibana_admin',
-        'test_logstash_reader',
-        'kibana_sample_admin',
-      ]);
+      await security.testUser.setRoles(
+        ['kibana_admin', 'test_logstash_reader', 'kibana_sample_admin'],
+        false
```

Comment:
no longer need a refresh?

Reply:
Well this was mostly @LeeDr 's idea.  Do you think we should take it out?

Reply:
@dmlemeshko perhaps we should take it out and re-run and see what happens.  whatcha think?

Reply:
no need to refresh, if it is working without it :)
maybe check git history to exclude it was made for Cloud run

Reply:
why couldn't we put `initTests()` here?

Reply:
Do you mean declare the `initTests()` fn here or call it here?

Reply:
We can put it wherever you'd like honestly :)

Reply:
you put it in `before` of child `describe` fns, can we do it only one time in parent?

Reply:
I can explain the refresh: true/false issue on setRoles a bit.

The default behavior of setRoles is to refresh the browser page after changing the roles and then wait for the chrome element.  The thought at the time setRoles was created was that you would be in some app in Kibana and change the roles and so we should refresh the browser to update the UI based on the new roles.

But refreshing the browser in this method can fail in several ways;

1. When you run a test by itself and the test hasn't navigated anywhere yet.  The test will fail because it tries to find that kibanaChrome element and it will never appear.
2. when the previous test ends on an app that the new roles don't have privs for.  For example, if an ML test just ran and left you on an ML page, and this test changes roles for a visual test which doesn't have privs for ML, when the browser refreshes it can fail to render kibanaChrome so the setRoles call fails.

So the simple and fast solution to both of those issues is to setRoles with refresh:false, and then navigate.